### PR TITLE
Preserve directory path of included file in Faucet config

### PR DESCRIPTION
**Pull request by didovesei**
For example, if the included file is `foo/bar/uniform.yaml`, with this PR, Forch will write it to `/etc/faucet/foo/bar/uniform.yaml`. Currently the output path is `/etc/faucet/uniform.yaml`.

### CODE REVIEW

File: forch/utils.py [hunk 1]
```diff
@@ -89,3 +90,12 @@ def dict_proto(message, proto_func, ignore_unknown_fields=False):
 def str_proto(message_text, proto_func):
     """Convert a string represented protobuf message to proto object"""
     return text_format.Parse(message_text, proto_func())
+
+
+def safe_mkdir(path):
+    try:
+        os.makedirs(path)
```
Comment (grafnu) [inline forch/utils.py:98]: How is this functionally different than

`os.makedirs(new_path, exist_ok=True)`

Comment (didovesei) [reply forch/utils.py]: ah, I didn't know there is such a parameter.. Updated

**Review by grafnu — COMMENTED** [review]
Lots of failing tests but the code itself looks ok...

**Comment by didovesei** [comment]
> Lots of failing tests but the code itself looks ok...

I have fixed the tests. PTAL..

**Review by grafnu — APPROVED** [review]

**Pull request merged**